### PR TITLE
implemented loading data from server

### PR DESCRIPTION
**Pull request by Stambuliak**
Implemented method of loading phones from back server.

### CODE REVIEW

**Review by VenomMB — CHANGES_REQUESTED** [review]
This is not a completed task fully, because you also need to load phones from API.
image

**Review by VenomMB — CHANGES_REQUESTED** [review]
Add API function.

File: src/services/fetchPhones.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const BASE_URL = 'https://server-store-p1t7.onrender.com';
+
+// returns a promise resolved after a given delay
+function wait(delay: number) {
+  return new Promise(resolve => {
+    setTimeout(resolve, delay);
+  });
+}
```
Comment (Anna-Kuzmenko) [inline src/services/fetchPhones.ts:9]: Do we need this delay?

**Review by Anna-Kuzmenko — APPROVED** [review]
:fire:

**Pull request merged**